packages/b/Bijou-JS.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Bijou.js",
```

Comment:
The `name` property must match the filename (excl. `.json`), so this either needs to be `Bijou-JS`, or the file needs to be named to `Bijou.js.json` :)

Reply:
Done! It's `Bijou.js.json` now! (I didn't want to do that originally because I was worried it'd count as a JS file)